ppdet/modeling/anchor_heads/yolo_head.py

```diff
@@ -269,7 +278,147 @@ def get_loss(self, input, gt_box, gt_label, gt_score):
             losses.append(fluid.layers.reduce_mean(loss))
             downsample //= 2
 
-        return sum(losses)
+        return {'loss': sum(losses)}
+
+    def get_splited_loss(self, outputs, targets, gt_box):
```

Comment:
maybe create `losses` folder in `ppdet/modeling`,  put this loss in that folder.

Reply:
Done, Thanks!